plugins/hls-class-plugin/hls-class-plugin.cabal

```diff
@@ -0,0 +1,29 @@
+cabal-version:      2.2
+name:               hls-class-plugin
+version:            0.1.0.0
+synopsis:           Explicit imports plugin for Haskell Language Server
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Junyoung Clare Jang
+maintainer:         jjc9310@gmail.com
+category:           Development
+build-type:         Simple
+
+library
+  exposed-modules:  Ide.Plugin.Class
+  hs-source-dirs:   src
+  build-depends:    aeson
+                  , base
+                  , containers
+                  , haskell-lsp
+                  , hls-plugin-api
+                  , ghc
+                  , ghc-exactprint
+                  , ghcide
+                  , lens
+                  , shake
+                  , text
+                  , transformers
+                  , unordered-containers
+
+  default-language: Haskell2010
```

Comment:
Shouldn't this project define its own test suite?

Reply:
Hm... Nice catch. I will try that.

Reply:
It looks like most of plugin packages like hlint plugin/retrie plugin/... do not have their own test suites. Would it still be better to add one?

Reply:
Yes. The retrie plugin doesn't have a test suite because it has no tests. But look at the tactics plugin for an example.

Reply:
Hmm... how about handling this test transfer in a separate PR?

Reply:
@pepeiborra If you don't mind, I will merge this PR first and then try to move the tests in a separate PR.